packages/react/src/ReactNodeViewRenderer.tsx

```diff
@@ -193,13 +193,13 @@ export class ReactNodeView<
     }
 
     if (from <= pos && to >= pos + this.node.nodeSize) {
-      if (this.renderer.props.selected) {
+      if (this?.renderer?.props?.selected) {
```

Comment:
This seemed to temporarily fix the issue, but also causes issues with other elements. Sandboxing did not work as requested in #5870 as the issue seems to disappear. Working on another solution in the meantime, would love to hear some suggestions, but I see how it could be difficult considering the reproducibility :)